xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-ui/src/main/resources/Refactoring/Code/RefactoringConfiguration.xml

```diff
@@ -41,25 +41,23 @@
 
 #set ($configurationObject = 'Refactoring.Code.RefactoringConfiguration')
 
-{{html clean='false'}}
+{{html clean='false' wiki='true'}}
```

Comment:
Why do you need wiki=true? You don't use wiki syntax here, do you?

Reply:
Indirectly I do, the result of `$doc.display('isRecycleBinSkippingActivated', 'edit')` is wrapped `{{html}}...{{/html}}` tags.

Reply:
Ah my bad I missed the fact that you don't have double html macro around `$doc.display` anymore.